src/performance_counters/register_counters.cpp

```diff
@@ -41,7 +43,14 @@ namespace phylanx { namespace performance_counters
             value.status_ = hpx::performance_counters::status_new_data;
             value.count_ = ++invocation_count_;
 
-            /*value.values_ = std::move(result);*/
+            std::vector<std::int64_t> result;
+            // Extract the values from instances_
```

Comment:
You might want to reserve the required space in this vector (`result.reserve(instances_.size());`)

Reply:
Done.